setuptools/tests/test_sdist.py

```diff
@@ -449,6 +449,20 @@ def test_sdist_with_latin1_encoded_filename(self):
             except UnicodeDecodeError:
                 filename not in cmd.filelist.files
 
+    def test_pyproject_toml_in_sdist(self):
```

Comment:
Should we also add a test that it can be successfully *excluded*?

Reply:
Yes, though that may prove more difficult to test.

Reply:
Turns out it wasn't hard at all. Fixed in f171cde1.